client/src/components/Header/Header.js

```diff
@@ -1,10 +1,32 @@
-import React from "react";
+import React, { Component } from "react";
 import { NavLink } from "react-router-dom";
 import NavItem from "./NavItem";
 import "./header.scss";
-import categories from "../../data/categories";
 
-class Header extends React.Component {
+class Header extends Component {
```

Comment:
Is there a benefit to this, or just something you prefer?

Reply:
There's no benefit besides not having to write `React.Component`